gdbgui-mpi/__main__.py

```diff
@@ -0,0 +1,15 @@
+from gdbgui import backend
```

Comment:
is this new file needed or can the existing `cli.py` be used?

Reply:
Probably is removable